corehq/apps/app_manager/models.py

```diff
@@ -1477,13 +1485,13 @@ def _get_media_by_language(self, media_attr, lang, strict=False):
 
     @property
     def default_media_image(self):
-        # For older apps that were migrated
-        return self.icon_by_language('default')
+        # For older apps that were migrated: just return the first available item
+        return self.icon_by_language('')
```

Comment:
These two functions should only be called in apps where `app.enable_localized_menu_media` is `False`, right? In which case the key `'default'` will always exist. Since you've stated you're not quite sure what's going on/how apps get in a particular state, maybe you should add a `soft_assert` here that `'default'` is the only language? That way you'll get emailed with examples of apps that aren't in the state you'd expect or examples of places calling these functions without first checking `enable_localized_menu_media`, or other weird/wrong states. Not blocking, just a thought.